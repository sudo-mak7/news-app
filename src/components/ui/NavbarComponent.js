import React from 'react'
import { Header, Menu } from 'semantic-ui-react'

const NavbarComponent = () => {
  return (
    <Menu fixed='top' widths={ 1 }>
      <Menu.Item as='a' header>
        <Header as='h1' textAlign='center'>LATEST NEWS</Header>
      </Menu.Item>
    </Menu>
  )
}

export default NavbarComponent