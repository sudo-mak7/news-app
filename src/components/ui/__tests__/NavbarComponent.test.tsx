import * as React from 'react'
import { render, screen } from '@testing-library/react'
import NavbarComponent from '@components/ui/NavbarComponent'

const navbarComponent = (): JSX.Element => <NavbarComponent />

describe('NavbarComponent test', () => {
  test('renders NavbarComponent', () => {
    render(navbarComponent())
    expect(navbarComponent()).toMatchSnapshot()

    const headerOfNavbarComponent = screen.getByRole('heading')
    expect(headerOfNavbarComponent).toBeInTheDocument()
  })
})