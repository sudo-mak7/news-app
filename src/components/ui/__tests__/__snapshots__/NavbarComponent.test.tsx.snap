// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarComponent test renders NavbarComponent 1`] = `<NavbarComponent />`;
